src/i18n/locales/de-DE.json

```diff
@@ -44,8 +44,7 @@
     },
     "save": {
         "headline-text": "Aktuelles Projekt speichern",
-        "confirm-button": "Ja, Änderungen speichern",
-        "confirmButton-default-tooltip": "Speichern-Button",
+        "confirm-button": "Ja, meine Änderungen verwerfen",
```

Comment:
This translation is not correct and unnecessary?

Reply:
This translation is wrong and is updated via Crowdin.